assets/app/views/create/fromimage.html

```diff
@@ -52,7 +60,7 @@
 
                 <div class="form-group">
                   <label for="sourceUrl" class="required">Git Repository URL</label>
-                  <div ng-class="{'has-warning': form.sourceUrl.$touched && !sourceURLPattern.test(buildConfig.sourceUrl), 'has-error': (form.sourceUrl.$error.required && form.sourceUrl.$touched)}">
+                  <div ng-class="{'has-warning': form.sourceUrl.$touched && !sourceURLPattern.test(buildConfig.sourceUrl), 'has-error': (form.sourceUrl.$error.required && form.sourceUrl.$dirty)}">
```

Comment:
so we only warn for the pattern after tabbing out of the field? what if the sample url is malformed (we dont control other peoples templates)  do you get the warning after clicking try it now?